frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -36,6 +36,227 @@ const connector = connect(
+
+  :hover {
+    cursor: pointer;
+  }
+
+  :active {
+    opacity: 0.8;
+  }
+`;
+
+const DashboardContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  min-height: 100%;
+  min-width: 100%;
+  padding-top: 98px;
+
+  @media (max-width: 900px) {
+    flex-direction: column;
+    padding-right: 0px;
+  }
+`;
+
+const MenuContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  min-height: 100%;
+  flex: 1;
+  border-radius: 0px 12px 0px 0px;
+
+  padding: 64px 64px 64px 64px;
+
+  background: ${CLOUD};
+
+  @media (max-width: 900px) {
+    width: 100%;
+    min-height: 347px;
+    border-radius: 0px 0px 0px 0px;
+    padding: 48px 24px 24px 24px;
+  }
+`;
+
+const MenuButtonContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 16px;
+
+  background: red;
+  width: 100%;
+  height: 100px;
+`;
+
+const TitleText = styled.p`
+  font-family: "Rubik";
+  font-size: 26px;
+  font-weight: 700;
+  text-transform: lowercase;
+  word-wrap: break-word;
+
+  @media (max-width: 900px) {
+    font-size: 30px;
+  }
+`;
+
+const MenuTopDiv = styled.div`
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+`;
+
+const ProfilePicture = styled.img`
+  margin-bottom: 24px;
+
+  height: 183px;
+  width: 183px;
+
+  @media (max-width: 900px) {
+    font-size: 30px;
+    height: 148px;
+    width: 148px;
+  }
+`;
+
+const SessionContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+
+  flex: 3;
+  min-height: 100%;
+  padding-left: 64px;
+  padding-top: 64px;
+  padding-bottom: 64px;
+
+  @media (max-width: 900px) {
+    min-width: 100%;
+    padding-top: 48px;
+    padding-right: 24px;
+    padding-left: 24px;
+  }
+`;
+
+const NextSessionButton = styled(PurpleButton)`
+  min-width: 32%;
+  height: 100px;
+  margin-bottom: 64px;
+  text-transform: lowercase;
+
+  @media (max-width: 900px) {
+    min-width: 100%;
+  }
+`;
+
+const WeekProgressContainer = styled.div`
+  flex: 2;
+  min-height: 100%;
+
+  padding-left: 64px;
+  padding-right: 64px;
+  padding-top: 64px;
+
+  @media (max-width: 900px) {
+    min-width: 100%;
+    padding-top: 48px;
+    padding-right: 24px;
+    padding-left: 24px;
+  }
+`;
+
+const ProgressStatsContainer = styled.div`
+  display: flex;
+  flex: 1;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  min-width: 100%;
+`;
+
+const ProgressBox = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 30px 30px 30px;
+
+  flex: 1 0 50%;
+  order: 0;
+  flex-grow: 1;
+
+  max-width: calc(50% - 16px);
+  margin-bottom: 32px;
+
+  background: ${CLOUD};
+  border-radius: 12px;
+
+  @media (max-width: 900px) {
+    max-width: calc(50% - 12px);
+    margin-bottom: 24px;
+  }
+`;
+
+const ProgressStatNumber = styled.p`
+  font-family: "Rubik";
+  font-size: 56px;
```

Comment:
![Screen Shot 2021-03-07 at 2 48 40 PM](https://user-images.githubusercontent.com/45154128/110252638-5bdb2100-7f54-11eb-95c5-fae83a33274a.png)
design nit: since like the words are kinda crammed in a box when resizing it, we can use 5vw (or anything along that line) for the font-size instead of 56px :))

Reply:
tyty